tests/js/server/replication/sync/replication-sync.js

```diff
@@ -861,6 +861,10 @@ function BaseTestConfig () {
     testSyncView: function () {
       connectToMaster();
 
+      // create custom analyzer
+      let analyzers = require('@arangodb/analyzers');
+      let analyzer = analyzers.save('custom', 'delimiter', { delimiter: ' ' }, ['frequency']);
```

Comment:
This test doesn't clean up after itself, i.e. it leaves the analyzer in place for any follow up tests. Currently this is not an issue, but in general tests should clean up.
I suggest we use something like
```
let analyzer = ...
try { 
  // test here
} finally {
  analyzers.remove(...) // clean up here
}
```